packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -47,22 +47,20 @@ export default {
   watch: {
     $route(to, from) {
       // On route change check if base path has changed.
-      // If true update `iHaveChildrenActive` parameter.
-      // In such way will be possible to indicate current active item without needs to re-render sidebar
+      // If true re-render sidebar.
+      // This condition will check if it's 'real' route change or just on page scroll
+      // In porccess when the page scrolling happens,
+      // Route could change it hash fragment and it also will trigger sidebar re-rendering
+      // But it shouldn't happen because page remains the same
```

Comment:
This comment is very helpful but could be a little bit clearer:
```suggestion
      // If true, re-render sidebar.
      // We want to check if it's a 'real' route change (re-render sidebar) or just a page scroll
      // where the hash fragment changes (do nothing)
```